src/components/MusicInfo.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div v-bind:id="'music-' + music.id " style="margin-bottom: 1rem;">
+    <md-card>
+      <md-card-header>
+        <span class="md-title">{{ music.music_name }}</span>
+      </md-card-header>
+      <md-card-content>
+        <p>Artist ID: {{ music.artist_id }}</p>
+        <p>Difficulty: {{ music.difficulty }}</p>
+      </md-card-content>
+      <md-card-actions>
+        <button class="md-button"
+                v-on:click="$emit('delete', index)">
+          <md-icon>delete</md-icon> &nbsp; <span>Delete</span>
+        </button>
+        <!--<button class="md-button">
```

Comment:
Please either uncomment or replace this commented code with a TODO(Tianyun): add Favor button.

Reply:
Currently we don't need to implement this button and favor function. These commented lines are now removed.